alot/commands/envelope.py

```diff
@@ -386,6 +386,7 @@ def openEnvelopeFromTmpfile():
                 del self.envelope[key]
 
             for value in vlist:
+                logging.debug('key: %s value [%s]', key, value)
```

Comment:
It helped me see which keys were set to None. I will remove it I think

Reply:
If some of these values are "None" then something went wrong (no pun intended). Setting headers to value "None" makes litte sense and those header should rather be removed IMO